template/5-mvp.md

```diff
@@ -2,68 +2,92 @@
+3. "As a frequent user, I want to be rewarded for my interactions and exploration."
 
-User Stories for Frequent Social Media Users:
+#### User Stories for Travelers:
+1. "As a traveler, I want to find unique local experiences by viewing posts from different cities."
+2. "As a traveler, I want to use a map to navigate to posts and challenges, so I can find my way easily in new locations."
 
-- "As a social media enthusiast, I want to discover posts around me so that I can explore new places and share my experiences."
-- "As a user, I want to create posts with a title, description, and location to share interesting places I visit."
-User Stories for Travelers:
-- "As a traveler, I want to find unique local experiences by viewing posts from different cities."
-User Stories for Local Explorers:
-- "As a local explorer, I want to discover hidden gems in my city by exploring posts from other residents."
+#### User Stories for Local Explorers:
+1. "As a local explorer, I want to discover and hidden gems in my city by exploring posts from other residents."
+2. "As a local resident, I want to create local challenges that encourage people to explore lesser-known areas of my city."
 
-*Identify and prioritise the key features required.*
-*Justify the importance of each feature.*
-
-- User Authentication: Necessary for security and personalized user experiences.
-- Post Creation: Central to the app's purpose of sharing and discovering locations.
-- Location-Based Feed: Key to the unique value proposition of Proxima, encouraging physical exploration.
-- Commenting System: Enhances user interaction and engagement.
-- Offline Mode: Ensures usability even without internet access, improving user retention.
-- Voting System: Helps surface the best content and enhances user experience.
-- Posts and comments management: Allows users to track and manage their activity.
-- Centauri Points: Adds a competitive and rewarding element, increasing user engagement.
-- Challenges: Drives user activity and exploration, fostering a dynamic user community.
-- Map Integration: Allows users to view posts on a map and navigate to specific locations.
+#### Key Features:
+- **User Authentication:** Necessary for security and personalized user experiences.
+- **Post Creation and Location-Based Feed:** The core functionality of sharing and discovering places.
+- **Interaction Tools (Comments and Voting):** Enhances user interaction and engagement.
+- **Centauri Points and Challenges:** Motivate continued app usage by creating challenges with rewards in form of points
+- **Ranking System:** Encourages competition and engagement by displaying user rankings based on Centauri points.
+- **Map:** Allows users to navigate to post locations and to visualize nearby activities.
+- **Posts and comments management:** Allows users to track and manage their activity.
```

Comment:
I wonder if we could make the title a little more explicit to show that we are taking of the posts/comments that the user has posted. Maybe something like this:

```suggestion
- **Users posts and comments management:** Allows users to track and manage their activity.
```

What do you think ?

Reply:
Thanks, applied in 071c6a9eae070ac2a327871d74ab561034ca096b